cilium/cmd/status.go

```diff
@@ -58,7 +59,7 @@ func init() {
 	statusCmd.Flags().BoolVar(&allRedirects, "all-redirects", false, "Show all redirects")
 	statusCmd.Flags().BoolVar(&allClusters, "all-clusters", false, "Show all clusters")
 	statusCmd.Flags().BoolVar(&brief, "brief", false, "Only print a one-line status message")
-	statusCmd.Flags().BoolVar(&verbose, "verbose", false, "Equivalent to --all-addresses --all-controllers --all-nodes --all-health")
+	statusCmd.Flags().BoolVar(&verbose, "verbose", false, "Equivalent to --all-addresses --all-controllers --all-nodes --all-health --all-redirects --all-clusters")
```

Comment:
minor nit, now that there's a `StatusDetails` object, I wonder if we could do `StatusAllDetails.String()` and have that object print each of these. Then if we expand it any further, we should extend `StatusAllDetails()` too?

Not a big deal though, even if we forget to update this help message it doesn't matter so much.

Reply:
Good point, will implement `StatusDetails.String()`.

Reply:
I decided against adding `StatusDetails.String()` since it would move CLI specific flag names to `pkg/client` which is not only used by CLI. Maybe we could just adjust the flag usage description to say something like `Verbose output. Includes details about certain status aspects` and no longer specify the flags it implies.